### ISSUE TITLE
Sudoku: inputs in game grid have no labels

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#315]
This is the new Sudoku at https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/sudoku/

All inputs in the grid have no label. 

Probably best to use aria-label in this case. Not sure what label should be as I don't play the game. Probably explaining what you have to type with a short label. Maybe it should say grid position so user doesn't get lost? Feel like need someone that plays game to give good suggestion on what context is needed to fill out the cell. Maybe the table structure will be enough to not need that context.
